docs/api-docs/orders/orders-api-overview.md

```diff
@@ -308,7 +308,7 @@ Accept: application/json
 
 > ### Note
 > * Custom products do not get added to the catalog.
-> * If price is not specified, the store's product catalog price is used; override this price with `price_inc_tax` and `price_ex_tax`.
+> * If a price is not specified, and you use the store's product catalog price; override this price with the `price_inc_tax` and `price_ex_tax`.
```

Comment:
I think the proposed change altered the meaning of that sentence. Perhaps, we can rewrite it to make it more comprehensible?
```suggestion
> * If you do not specify the product price in the [create order](https://developer.bigcommerce.com/api-reference/store-management/orders/orders/createanorder request, the product's catalog price will be used. You can override this price with the `price_inc_tax` and `price_ex_tax`.
```